src/app/containers/StoryPromo/MediaIndicator/index.jsx

```diff
@@ -31,7 +31,7 @@ const getCpsMediaTypes = item => {
   const hasMediaInfo = path(['media'], item);
 
   // Only build a media indicator if this is a photo gallery or media item
-  if (!isPGL && (!isCpsMedia || !hasMediaInfo)) {
+  if (!isPGL && !(isCpsMedia || hasMediaInfo)) {
```

Comment:
These two booleans are not quite equivalent

https://www.wolframalpha.com/input/?i=%21a+%26%26+%28%21b+%7C%7C+%21c%29
https://www.wolframalpha.com/input/?i=%21a+%26%26+%21%28b+%7C%7C+c%29

(ignore me if this is an intentional logic change and not just a refactor 😛)

Reply:
😅 It's an intentional logic change. The initial logic  was incorrect because if `isCpsMedia` was false, `!isCpsMedia` will always return true and hasMediaInfo will not be calculated. I think it will be clearer when I push the tests

Reply:
What if we removed the `isCpsMedia` check altogether and just rely on the `hasMediaInfo` part. We can assume that if cpsType=MAP, then it should always have media. And if it doesn't, it won't pass the `hasMediaInfo` check anyway